lib/ringbuf/src/lib.rs

```diff
@@ -235,6 +231,24 @@ pub struct RingbufEntry<T: Copy + PartialEq> {
     pub payload: T,
 }
 
+/// A container for a `Ringbuf` at static scope, which includes a `StaticCell`
+/// to access it safely, and an enable flag.
+pub struct StaticRingbuf<T: Copy + PartialEq, const N: usize> {
+    pub ringbuf_cell: StaticCell<Ringbuf<T, N>>,
+    pub enable: AtomicBool,
```

Comment:
Getting greedy here, but how difficult would it be for us to locate this address from Humility? It would be incredible to be able to turn this on dynamically (and apologies if you were thinking in this direction all along!).